### DISCUSSION TITLE
How to version the configurations

### DISCUSSION THREAD

**Discussion by AleixMT** [linuxserver/Heimdall#1071] [Q&A]
I configured a Heimdall instance with my apps and links. I am using `docker-compose`.

Now I would like to save that configuration in order to be able to restore my Heimdall installation easily. I was thinking in using a `git` repository to store all the relevant changes or keeping a backup of the `config` folder of the Heimdall instance.

What would be the recommended way to save the configuration that I have been doing? What would be the way to retore it? I have preference in using a `git` repository to version the configuration.

Sorry if this is a stupid question or it has already been explained. Thank you.

**Comment by derrick-anderson** [comment]
I would also love to understand where the data that handles the config is stored? I've read through the config files but don't yet understand how you would update a heimdall installation without losing all your setup?

**Reply by AleixMT** [reply]
As you can tell by seeing that I am the OP of this issue, I am not an expert regarding this. But I do know that you need to save the `/config` folder from inside the container, upgrade the image and rerun with the new image and the `/config` folder with the same data before the upgrade.
